ngrinder-frontend/src/js/components/perftest/detail/Config.vue

```diff
@@ -428,21 +431,28 @@
             openedWindow.focus();
         }
 
+        openingScriptSelect() {
+            if (this.gitHubLazyLoading && this.isGitHubStorage()) {
+                this.LoadGitHubScript();
+            }
+        }
+
         changeScript(revision) {
-            if (this.test.config.scm === 'svn') {
-                if (this.$refs.scriptSelect.getSelectedOptionValidate() !== '-1') {
-                    this.test.config.scriptRevision = revision;
-                    this.getTargetHosts();
-                    this.getScriptResource();
-                    this.display.showRevisionBtn = true;
-                } else {
-                    this.test.config.scriptRevision = -1;
-                    this.targetHosts = [];
-                    this.resources = [];
-                    this.display.showRevisionBtn = false;
-                }
-            } else {
+            if (this.isGitHubStorage()) {
                 this.test.config.scriptRevision = revision;
+                return;
+            }
+
+            if (this.$refs.scriptSelect.getSelectedOptionValidate() !== '-1') {
+                this.test.config.scriptRevision = revision;
+                this.getTargetHosts();
+                this.getScriptResource();
+                this.display.showRevisionBtn = true;
+            } else {
+                this.test.config.scriptRevision = -1;
```

Comment:
scriptRevision should be '-1' now?

Reply:
When change script, `svn` script always set to `HEAD` revision.